frontend/apps/reader/modules/readermenu.lua

```diff
@@ -53,7 +53,6 @@ function ReaderMenu:init()
     self.registered_widgets = {}
 
     if Device:hasKeys() then
-        self.key_events = { Close = { { "Back" }, doc = "close menu" }, }
```

Comment:
I'm so used to Backspace to quit things on the emulator, which work(ed) on quite everything.
No real way to keep that while staying ok with what you want?